src/utils/filterContests.js

```diff
@@ -3,14 +3,18 @@ import type { Contest } from '../types';
 
 export function filterContests(contests: Array<Contest>, from?: Date, to?: Date): Array<Contest> {
```

Comment:
`filterContests` is a vague name. I suggest `filterContestsByDate` or something similar.

Reply:
also, if people want to filter only by `startTime`, which is probably going to be used a lot, they will have to pass `null` or similar every time?

to avoid this, create a wrapper for this function like `filterContestsStartingFrom` and just call this function with the correct arguments